### PR TITLE
[Script] Release helper

### PR DESCRIPTION
**Pull request by BigCat20196**
add multi policy
https://github.com/Azure/azure-sdk-for-python/issues/25123

test link:
https://github.com/Azure/sdk-release-request/issues/2987

### CODE REVIEW

File: scripts/release_helper/python.py [hunk 1]
```diff
@@ -41,10 +44,23 @@ def init_readme_link(self) -> None:
...
 
+    def check_tag_consistency(self) -> None:
+        self.target_readme_tag = self.target_readme_tag.replace('tag-', '')
+        super().check_tag_consistency()
+        if self.default_readme_tag != self.target_readme_tag:
+            self.add_label(_INCONSISTENT_TAG)
+
+    def multi_api_policy(self) -> None:
+        if self.is_multiapi:
+            self.bot_advice.append('It is a MultiAPI')
```
Comment (msyyc) [inline scripts/release_helper/python.py:60]:
```suggestion
            self.bot_advice.append('MultiAPI')
```

File: scripts/release_helper/python.py [hunk 2]
```diff
@@ -41,10 +44,23 @@ def init_readme_link(self) -> None:
...
+        self.is_specified_tag = any('->Readme Tag:' in line for line in issue_body_list)
 
         # get readme_link
         self.get_readme_link(origin_link)
 
+    def check_tag_consistency(self) -> None:
+        self.target_readme_tag = self.target_readme_tag.replace('tag-', '')
+        super().check_tag_consistency()
+        if self.default_readme_tag != self.target_readme_tag:
+            self.add_label(_INCONSISTENT_TAG)
```
Comment (msyyc) [inline scripts/release_helper/python.py:56]: This could be a common policy for all languages

File: scripts/release_helper/python.py (hunk 1 above)
Comment (msyyc) [reply scripts/release_helper/python.py:60]: If there is already `auto-ask-check`, it proves that we have handled it and then there is no need to hint it.

File: scripts/release_helper/python.py [hunk 3]
```diff
@@ -53,10 +69,11 @@ def get_package_and_output(self) -> None:
         pattern_output = re.compile(r'\$\(python-sdks-folder\)/(.*?)/azure-')
         self.package_name = pattern_package.search(contents).group().split(':')[-1].strip()
         self.output_folder = pattern_output.search(contents).group().split('/')[1]
-        self.is_multiapi = ('MultiAPI' in self.issue_package.labels_name) or ('multi-api' in contents)
+        self.is_multiapi = (_MULTI_API in self.issue_package.labels_name) or ('multi-api' in contents)
 
     def get_edit_content(self) -> None:
-        self.edit_content = f'\n{self.readme_link.replace("/readme.md", "")}\n{self.package_name}'
+        self.edit_content = f'\n{self.readme_link.replace("/readme.md", "")}\n{self.package_name}' \
+                            f'\nReadme Tag: {self.target_readme_tag}'
```
Comment (msyyc) [inline scripts/release_helper/python.py:76]: Why do we need a new line?

Comment (BigCat20196) [reply scripts/release_helper/python.py:76]: > Why do we need a new line?

Because it exceeds 120 characters.

**Review by msyyc — APPROVED** [review]

**Pull request merged**